jena-arq/src/main/java/org/apache/jena/riot/out/JsonLDWriter.java

```diff
@@ -98,10 +96,8 @@ private void serialize(Writer writer, DatasetGraph dataset, PrefixMap prefixMap,
             // Unclear as to the way to set better printing.
             obj = JsonLdProcessor.compact(obj, localCtx, opts) ;
 
-            if ( isPretty() )
-                JsonUtils.writePrettyPrint(writer, obj) ;
-            else
-                JsonUtils.write(writer, obj) ;
+            if ( isPretty() ) JsonUtils.writePrettyPrint(writer, obj) ;
+            else JsonUtils.write(writer, obj) ;
```

Comment:
I don't like if statements with their positive clause on the same line unless they have no else clause and are not part of the flow of the logic. (e.g. not part of the flow: `if ( DEBUG) System.err.println(....) ;`)